### DISCUSSION TITLE
How to make it work with powerline

### DISCUSSION THREAD

**Discussion by samueloph** [rickstaa/tmux-notify#56] [Q&A]
I use https://github.com/powerline/powerline on my shell and I didn't manage to make it work even with `set -g @tnotify-prompt-suffixes ''`, I must be missing something obvious.

```
echo $PS1
\[\] samueloph \[\] \[\]~ \[\] \[\]
```

I have also tried it with the values `]` and `\]` but it didn't work.

I've confirmed that I can send notifications manually with `notify-send "test"`

**Comment by rickstaa** [comment]
Hey @samueloph, Thank you for your interest in our plugin! While I haven't personally tested it with powerline, it should be compatible with most shells once you've determined the correct syntax. You can find a detailed answer on how to find the right syntax for your shell in this GitHub issue: [Link to issue](https://github.com/rickstaa/tmux-notify/issues/22#issuecomment-1691319391). Please give it a try, and feel free to let me know if it resolves your issue 👍🏻.
